packages/office-ui-fabric-react/src/components/ThemeGenerator/ThemeRulesStandard.ts

```diff
@@ -39,8 +39,8 @@ export enum FabricSlots {
   neutralQuaternary, // BaseSlots.backgroundColor, Shade[Shade.Shade5]);
   neutralTertiaryAlt, // BaseSlots.backgroundColor, Shade[Shade.Shade6]); // bg6 or fg2
   neutralTertiary, // BaseSlots.foregroundColor, Shade[Shade.Shade3]);
-  // deprecated: neutralSecondaryAlt, // BaseSlots.foregroundColor, Shade[Shade.Shade4]);
-  neutralSecondary, // BaseSlots.foregroundColor, Shade[Shade.Shade5]);
+  neutralSecondaryAlt, // BaseSlots.foregroundColor, Shade[Shade.Shade5]);
+  neutralSecondary, // BaseSlots.foregroundColor, Shade[Shade.Shade4]);
```

Comment:
is this intentional?

Reply:
Yes Shade5 in the comment of neutralSecondary doesn't match the original setting anyways (which is Shade4 down below on line 132) so I corrected it to Shade4